### PR TITLE
 #207 Updated inscriptions font

### PR DESCRIPTION
**Pull request by NikitaSukhanov**

### CODE REVIEW

File: app/src/main/res/layout-land/fragment_columns.xml [hunk 1]
```diff
@@ -81,39 +80,28 @@
...
             <Button
                 android:id="@+id/button_remove"
                 android:layout_width="35dp"
                 android:layout_height="35dp"
-                android:layout_marginBottom="4dp"
                 android:background="@null"
                 android:drawableStart="@drawable/ic_remove"
                 app:layout_constraintBottom_toTopOf="@+id/switch_invert"
                 app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintHorizontal_bias="0.51"
```
Comment (maik-nack) [inline app/src/main/res/layout-land/fragment_columns.xml]: Why do we need it?

Comment (NikitaSukhanov) [reply app/src/main/res/layout-land/fragment_columns.xml]: I don't know, it was in previous version in several places, i decided not to touch it.
I may remove it, if necessary.

File: app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentColumns.java [hunk 2]
```diff
@@ -293,56 +294,54 @@ void setCustomImageViewSizes(int width, int height) {
...
         }
+
+        int maxColumns = (_countColumns + remainder) > 90
+                ? 90 : (_countColumns + remainder);
+        maxColumns = maxColumns > width ? width : maxColumns;
+        int minColumns = (_countColumns - remainder) < 5
+                ? 5 : (_countColumns - remainder);
+        int maxRowsRound = (int) (maxColumns * _coefBitmap);
+
+        return context.getString(stringID,
```
Comment (maik-nack) [inline app/src/main/java/com/mithridat/nonoconverter/ui/editimage/FragmentColumns.java]: `context` is optional.

**Review by maik-nack — APPROVED** [review]
Great job!

**Review by PawelMelnik — APPROVED** [review]
Good job!

**Pull request merged**